tools/pg_inspector/dump_server.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+basedir=$(dirname $0)
+export PGPASSWORD="${PGPASSWORD:-smartvm}"
+
+${basedir}/../pg_inspector.rb servers
```

Comment:
this script should probably live in https://github.com/ManageIQ/manageiq-appliance/tree/master/LINK/usr/bin.  Maybe call it `pg_inspector_servers.rb`